examples/bad_request/bad_request.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/asciimoo/colly"
+)
+
+func main() {
+	c := colly.NewCollector()
+
+	c.AllowedDomains = []string{"definitely-note-a-website.not"}
+
+	c.OnHTML("a[href]", func(e *colly.HTMLElement) {
```

Comment:
Wouldn't be this example more clean and understandable without the `OnHTML` callback?

Reply:
Yeah, I just kept this in here to show it wouldn't be called.